src/main/java/elizCurtnazar/driver/BrowserManager.java

```diff
@@ -0,0 +1,11 @@
+package elizCurtnazar.driver;
```

Comment:
We'll need to sort this conflict out before being able to advance. We'll take it in today's call.

Reply:
done